packages/rum/src/boot/startRecording.spec.ts

```diff
@@ -19,70 +18,62 @@ import { startRecording } from './startRecording'
 const VIEW_TIMESTAMP = 1 as TimeStamp
 
 describe('startRecording', () => {
-  let setupBuilder: TestSetupBuilder
+  const lifeCycle = new LifeCycle()
   let sessionManager: RumSessionManagerMock
   let viewId: string
   let textField: HTMLInputElement
   let requestSendSpy: jasmine.Spy<HttpRequest['sendOnExit']>
   let stopRecording: () => void
-  let clock: Clock | undefined
   let configuration: RumConfiguration
 
-  beforeEach(() => {
-    if (isIE()) {
-      pending('IE not supported')
-    }
-    configuration = {} as RumConfiguration
+  function setupStartRecording() {
+    configuration = {
+      defaultPrivacyLevel: DefaultPrivacyLevel.ALLOW,
+      applicationId: 'appId',
+    } as unknown as RumConfiguration
```

Comment:
**🥜 nitpick:** ‏`configuration` could be locally-scoped in `setupStartRecording()`

```ts
const configuration = {...}
```